feature-libs/product/configurators/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -0,0 +1,92 @@
+<div
+  class="cx-configurator-attribute-product-card"
+  [ngClass]="{
+    'cx-configurator-attribute-product-card-selected': product.selected
```

Comment:
maybe rename this to something shorter, like `selected` or `card-selected`

Reply:
i use bem-like naming convention so cx-configurator-attribute-product-card have cx-configurator-attribute-product-card-selected state

Reply:
changed